src/test/java/teammates/test/pageobjects/InstructorFeedbackSessionsPage.java

```diff
@@ -299,11 +298,11 @@ public void fillTimeValueIfNotNull(String dateId, LocalDateTime datetimeValue, W
      * passes consistently, do not try to click on the datepicker element using Selenium as it will
      * result in a test that passes or fail randomly.
     */
-    public void fillTimeValueForDatePickerTest(String timeId, Calendar newValue) {
+    public void fillTimeValueForDatePickerTest(String timeId, LocalDateTime newValue) {
```

Comment:
I think this is the only use case of `formatDateForSessionsForm` where a `LocalDate` instead of `LocalDateTime` is preferred, so I will keep the current signature of `LocalDateTime` and convert the `LocalDate` here.